src/helm/benchmark/model_metadata_registry.py

```diff
@@ -0,0 +1,70 @@
+"""This allows users to register additional models via configuration file.
+
+TODO(#1673): Support adding other kinds of models besides AI21.
+"""
+
+from typing import Any, Dict, Iterable, Optional, List
+from dataclasses import dataclass, field
+from datetime import date
+
+import cattrs
+import dacite
+import yaml
+from helm.benchmark.presentation.schema import ModelField
+
+from helm.common.general import parse_hocon
+from helm.proxy.models import ALL_MODELS, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, MODEL_NAME_TO_MODEL, TEXT_MODEL_TAG, Model
+
+
+@dataclass(frozen=True)
+class ModelMetadata:
+    name: str
+
+    # Organization that originally created the model (e.g. "EleutherAI")
+    #   Note that this may be different from group or the prefix of the model `name`
+    #   ("together" in "together/gpt-j-6b") as the hosting organization
+    #   may be different from the creator organization. We also capitalize
+    #   this field properly to later display in the UI.
```

Comment:
Not sure if we want to capitalize...because then we will have `Meta` and `meta`...  I don't think we should conflate unique names with display names...for example, the display name might have a space, and I don't think we want the creator_organization to have a space.

Reply:
In the version in `schema.yaml`, this was mixed case... maybe we should revisit later.